test/connection-pool.test.js

```diff
@@ -1,16 +1,13 @@
 var assert = require('assert');
 var Request = require('tedious').Request;
+var fs = require('fs');
 var ConnectionPool = require('../lib/connection-pool');
 
-var connectionConfig = {
-    userName: 'test',
-    password: 'test',
-    server: 'dev1',
-    options: {
-        appName: 'pool-test',
-        database: 'test'
-    }
-};
+function getConfig() {
```

Comment:
What's the reasoning for this? I seems like having the connection config in an external file that is not included in the project is unnecessarily convoluted.